keras_retinanet/utils/image.py

```diff
@@ -19,6 +19,12 @@
 import time
 import numpy as np
 import cv2
+import PIL
+
+
+def read_image_bgr(path):
+    image = np.asarray(PIL.Image.open(path).convert('RGB'))
+    return image[:, :, ::-1]
```

Comment:
We should avoid this (it might be responsible for #97), or at least make a copy afterwards.
This creates a view:
```
>>> image = np.asarray(PIL.Image.open(path).convert('RGB'))
>>> image.flags
  C_CONTIGUOUS : True
  F_CONTIGUOUS : False
  OWNDATA : False
  WRITEABLE : False
  ALIGNED : True
  UPDATEIFCOPY : False
>>> image = np.asarray(PIL.Image.open('crop.png').convert('RGB'))[:,:,::-1]
>>> image.flags
  C_CONTIGUOUS : False
  F_CONTIGUOUS : False
  OWNDATA : False
  WRITEABLE : False
  ALIGNED : True
  UPDATEIFCOPY : False
```

Reply:
I don't think that issue was caused by this because `master` currently still uses `cv2.imread`, but I get your point. @de-vri-es do you mind making the suggested adjustments? My preference would be to use `copy()`, since `opencv` does that internally anyway.